TASK/SetDriveTaskDrivingStyle.md

```diff
@@ -8,16 +8,81 @@ ns: TASK
 void SET_DRIVE_TASK_DRIVING_STYLE(Ped ped, int drivingStyle);
 ```
 
-```
-This native is used to set the driving style for specific ped.  
-Driving styles id seems to be:  
-786468  
-262144  
-786469  
-http://gtaforums.com/topic/822314-guide-driving-styles/  
+Sets the driving style for a ped currently performing a driving task.
+
+Each flag in the `VehicleDrivingFlags` enum can be combined to create a driving style, with each enabling or disabling a specific driving behavior. The driving style can be set to one of the predefined driving styles, or a custom driving style can be created by combining the flags. This can be done by using the bitwise OR operator (`|`) to combine the flags or by adding the decimal values of the flags together.
```

Comment:
```suggestion
Each flag in the `eVehicleDrivingFlags` enum can be combined to create a driving style, with each enabling or disabling a specific driving behavior. The driving style can be set to one of the predefined driving styles, or a custom driving style can be created by combining the flags. This can be done by using the bitwise OR operator (`|`) to combine the flags or by adding the decimal values of the flags together.
```